Album.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+class Album
+{
+    const OWNER = 'iO Academy';
+
+    // Properties
+    private int $id;
+    private string $name;
+    private string $release_date;
+    private string $album_or_single;
+    private ?string $link_to_image;
+    private string $reason_for_inclusion;
+    private ?string $artists;
+
+    public function __construct(
+        int $id, string $name, string $release_date, string $album_or_single, ?string $link_to_image,
+        string $reason_for_inclusion, ?string $artists
+    ) {
```

Comment:
When you have a long list of params like this in a method, the way to break it down is like this:

```
public function __construct(
    int $foo,
    float $bar,
    string $baz
) {

```

Makes it easier to read and change later